blog/2019-05-28-first-blog-post.md

```diff
@@ -0,0 +1,12 @@
+---
+slug: first-blog-post
+title: First Blog Post
+authors:
+  name: Gao Wei
+  title: Docusaurus Core Team
+  url: https://github.com/wgao19
+  image_url: https://github.com/wgao19.png
+tags: [hola, docusaurus]
+---
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque elementum dignissim ultricies. Fusce rhoncus ipsum tempor eros aliquam consequat. Lorem ipsum dolor sit amet
```

Comment:
Lorem ipsum?

Reply:
Oversight on my part, these template blog posts I originally intended to modify into our substack posts and came with the template docusaurus. We can add this later if we feel the need to.